dtcli/scripts/dt.py

```diff
@@ -389,6 +391,45 @@ def upload(**kwargs):
 
 
 
+@extension.command(
+    help="Downloads all schemas from choosen version"
+)
+@click.option(
+    "--version", prompt=True, help="Schema Version e.g. 1.235"
+)
+@click.option(
+    "--tenant-url", prompt=True, help="Dynatrace environment URL, e.g., https://<tenantid>.live.dynatrace.com"
+)
+@click.option(
+    "--api-token",  prompt=True, help="Dynatrace API token. Please note that token needs to have the 'Write extension' scope enabled."
+)
+@click.option(
+    "--download-dir",
+    default=DEFAULT_DOWNLOAD_DIR, show_default=True,
+    help="Directory where folder schemas will be created with all downloaded files",
+)
+def schemas(**kwargs):
+    dt = api.DynatraceAPIClient.from_tenant_url(kwargs['tenant_url'], kwargs['api_token'])
+    dt.download_schemas(kwargs['version'], kwargs['download_dir'])
+
+
+@extension.command(
+    help="Wipe out extension from Dynatrace Cluster e.g., custom:com.dynatrace.extension.extension-name"
+)
+@click.option(
+    "--extension", prompt=True, help="Extension name e.g., custom:com.dynatrace.extension.extension-name"
+)
+@click.option(
+    "--tenant-url", prompt=True, help="Dynatrace environment URL, e.g., https://<tenantid>.live.dynatrace.com"
+)
+@click.option(
+    "--api-token",  prompt=True, help="Dynatrace API token. Please note that token needs to have the 'Write extension' scope enabled."

```

Comment:
This isn't secure as the token will be stored in history

Reply:
This is a convention here in dt-cli. I make it like others option. It will be safer and more comfortable to get them from file.